aws/resource_aws_dlm_lifecycle_policy_test.go

```diff
@@ -0,0 +1,292 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/dlm"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSDlmLifecyclePolicyBasic(t *testing.T) {
+	resourceName := "aws_dlm_lifecycle_policy.basic"
+
+	resource.Test(t, resource.TestCase{
```

Comment:
Nit: This was written before `resource.ParallelTest()` was available so we'll update these. 👍